File: Clase6/SoftProgSolution/SoftProgModel/src/main/java/pe/edu/pucp/softprog/rrhh/model/Empleado.java
package pe.edu.pucp.softprog.rrhh.model;

import java.util.Date;

public class Empleado extends Persona{
    private Area area;
    private String cargo;
    private double sueldo;
    private boolean activo;

    public Empleado(){}

    public Empleado(String DNI, String nombre, String apellidoPaterno, char genero, Date fechaNacimiento, Area area, String cargo, double sueldo) {
        super(DNI, nombre, apellidoPaterno, genero, fechaNacimiento);
        this.area = area;
        this.cargo = cargo;
        this.sueldo = sueldo;
    }

    public String getCargo() {
        return cargo;
    }

    public void setCargo(String cargo) {
        this.cargo = cargo;
    }

    public double getSueldo() {
        return sueldo;
    }

    public void setSueldo(double sueldo) {
        this.sueldo = sueldo;
    }

    public boolean isActivo() {
        return activo;
    }

    public void setActivo(boolean activo) {
        this.activo = activo;
    }

    public Area getArea() {
        return area;
    }

    public void setArea(Area area) {
        this.area = area;
    }
}

File: Clase12/ProyectosNET/SoftProgSolution/SoftProgModel/RRHH/Empleado.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace SoftProgModel.RRHH
{
    public class Empleado : Persona
    {
        private Area _area;
        private String _cargo;
        private double _sueldo;
        private bool _activo;
        public Empleado() { }
        public Empleado(string DNI, string nombre, string apellidoPaterno, char genero, DateTime fechaNacimiento, string cargo, double sueldo) : base(DNI, nombre, apellidoPaterno, genero, fechaNacimiento)
        {
            _cargo = cargo;
            _sueldo = sueldo;
        }

        public string Cargo { get => _cargo; set => _cargo = value; }
        public double Sueldo { get => _sueldo; set => _sueldo = value; }
        public bool Activo { get => _activo; set => _activo = value; }
        public Area Area { get => _area; set => _area = value; }
    }
}
